tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/ModuleFragmentProjectProvider.java

```diff
@@ -72,7 +73,8 @@ public IStatus createNewProject( NewModuleFragmentOp op, IProgressMonitor monito
 
         final String hostBundleName = op.getHostOsgiBundle().content();
 
-        final IPath temp = GradleCore.getDefault().getStateLocation().append( hostBundleName );
+        final IPath temp = GradleCore.getDefault().getStateLocation().append(
```

Comment:
We are caching this content during project creation and it could change by the time the user opens a compare file.  So we should only find this content right before we open the compare.